src/views/Exchange/Exchange.js

```diff
@@ -173,8 +174,23 @@ export function getPositionQuery(tokens, nativeTokenAddress) {
 }
 
 export default function Exchange({ savedIsPnlInLeverage, setSavedIsPnlInLeverage, savedSlippageAmount, pendingTxns, setPendingTxns, savedShouldShowPositionLines, setSavedShouldShowPositionLines }) {
+  const [bannerHidden, setBannerHidden] = useState(false);
```

Comment:
if possible it's better to use `useLocalStorage` from 'react-use' like we do in other parts of project
example https://github.com/gmx-io/gmx-interface/blob/master/src/Helpers.js#L5

it will bond state to localstorage for you

Reply:
done